recipes/dynamic-network-architectures/meta.yaml

```diff
@@ -1,6 +1,5 @@
 {% set name = "dynamic-network-architectures" %}
 {% set version = "0.3.1" %}
-{% set python_min = "3.10" %}
```

Comment:
You actually need this as this recipe doesn't support 3.9, the global min

Reply:
@xhochy Thanks for the reminder - I updated the recipe, can you take another look?